### DISCUSSION TITLE
Disable physical monitors while streaming and enable when stream ends

### DISCUSSION THREAD

**Discussion by arthurzc** [ClassicOldSong/Apollo#180] [Q&A]
Hello!

I have switched from Sunshine (installed via sunshine AIO) to Apollo a week ago and so far I am in love with it. There is a single feature that I enjoyed so much in Sunshine (that used SudoVDA) is that it turned off my physical monitors automatically for me while streaming and re-enabled them when the stream ended.

Is there a way to do the same with Apollo? If so, how can I do it?

I have been searching for it but with no luck.

Thanks in advance!

**Comment by arthurzc** [comment]
Update: I was able to make it work by disabling the physical monitors directly in Windows Display Settings during the stream. Thanks for the help!

**Reply by ClassicOldSong** [reply]
That's the major difference between Apollo and other display swapping solutions: despite the creation and removal of virtual displays, it's natively managed by Windows!

Have fun!

**Comment by meanmrgreen** [comment]
I found a "simple" workaround if you could call it that.
A smartplug connected to a powerstrip with all my monitors plugged into and i turn that off before i connect. That will remove the displays from windows and force all programs to open on the virtual display.

Im betting there is somesort of uber windows command to disable displays but what i then did was to create a autohotkey script that sends a signal to my homeassistant to turn off the display. Then use this as a global start command.

**Comment by djusHa** [comment]
If you are on Windows and have some powershell skills, you can use this excellent Module: https://github.com/MartinGC94/DisplayConfig, to write your own script.

The Script(s) run then if a session starts or ends.

Let me know if you want an example Script.

**Comment by Snsv69** [comment]
Solution found: Use DisplaySwitch.exe /external as the Do command and DisplaySwitch.exe /internal as the Undo command in the Resume/Pause Commands section of the Desktop app in Apollo. No external script needed, it's a native Windows command.
